lib/cocoapods/target/pod_target.rb

```diff
@@ -343,7 +343,16 @@ def framework_paths
             dsym_source = if dsym_path.exist?
                             "${PODS_ROOT}/#{relative_path_to_sandbox}.dSYM"
                           end
-            FrameworkPaths.new(framework_source, dsym_source)
+            if framework_path.dirname.exist?
+              bcsymbolmap_paths = framework_path.dirname.glob('*.bcsymbolmap')
+              bcsymbolmap_paths.map! do |bcsymbolmap_path|
+                bcsymbolmap_relative_path_to_sandbox = bcsymbolmap_path.relative_path_from(sandbox_root)
+                "${PODS_ROOT}/#{bcsymbolmap_relative_path_to_sandbox}" if bcsymbolmap_path.exist?
```

Comment:
`if bcsymbolmap_path.exist?` needed here? Wouldn't `glob` only return files that exist?

Reply:
Thanks for your carefully review
Delete `if bcsymbolmap_path.exist?` later.
Map those paths to relative paths is needed.

Reply:
I do not understand. `if bcsymbolmap_path.exist?` is _not_ needed.